test_technical_interview_flow.py

```
@@ -0,0 +1,180 @@
+        )
+        if response.status_code != 200:
+            print(f"Error: {response.text}")
+            return
+
+    print("✅ 구조화 면접 완료")
+
+    # 2. Technical Interview 시작
+    print("\n[1단계] 직무 적합성 면접 시작")
+
+    # 모의 access_token (실제로는 JWT)
+    mock_token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxNiIsImVtYWlsIjoic2FuZ0BuYXZlci5jb20iLCJyb2xlIjoidGFsZW50IiwiZXhwIjoxNzYwMjcwMDI1fQ.O8ZBAxnuDYJIYrqp0NJJuqY8Z5T-K7KZ9pVwmcGw1Z0"
```

Ruff (0.13.3)

Possible hardcoded password assigned to: "mock_token"

(S105)